airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -12594,11 +12594,7 @@
             "type": "string",
             "title": "Reader Options",
             "description": "This should be a string in JSON format. It depends on the chosen file format to provide additional options and tune its behavior.",
-            "examples": [ "{}", "{\"sep\": \" \"}", {
-              "sep": "\t",
-              "header": 0,
-              "names": [ "column1", "column2" ]
-            } ]
+            "examples": [ "{}", "{\"sep\": \" \"}", "{\"sep\": \"\t\", \"header\": 0, \"names\": [\"column1\", \"column2\"] }" ]
```

Comment:
Check this one. It look like hided update for source file.

Reply:
These changes are introduced in https://github.com/airbytehq/airbyte/pull/25067/commits/dc707f729db2371552f87311b048b6087751bf9c and  made by [octavia-squidington-iii](https://github.com/airbytehq/airbyte/commits?author=octavia-squidington-iii)

Reply:
Let's revert those changes.

Reply:
Reverted;
Octavia has made another changes: https://github.com/airbytehq/airbyte/pull/25067/commits/9a1e09a0e734533d56fddbfa5fb96ce974e0d4ee

Let's leave it as is.